lib/kb_SetUtilities/kb_SetUtilitiesImpl.py

```diff
@@ -191,6 +669,150 @@ def __init__(self, config):
         pass
 
 
+    def KButil_Split_GenomeSet(self, ctx, params):
+        """
+        :param params: instance of type "KButil_Split_GenomeSet_Params"
+           (KButil_Split_GenomeSet() ** **  Method for evenly splitting
+           Genome Set) -> structure: parameter "workspace_name" of type
+           "workspace_name" (** The workspace object refs are of form: ** ** 
+           objects = ws.get_objects([{'ref':
+           params['workspace_id']+'/'+params['obj_name']}]) ** ** "ref" means
+           the entire name combining the workspace id and the object name **
+           "id" is a numerical identifier of the workspace or object, and
+           should just be used for workspace ** "name" is a string identifier
+           of a workspace or object.  This is received from Narrative.),
+           parameter "input_ref" of type "data_obj_ref", parameter
+           "split_num" of Long
+        :returns: instance of type "KButil_Split_GenomeSet_Output" ->
+           structure: parameter "report_name" of type "data_obj_name",
+           parameter "report_ref" of type "data_obj_ref"
+        """
+        # ctx is the context object
+        # return variables are: returnVal
+        #BEGIN KButil_Split_GenomeSet
+        console = []
+        invalid_msgs = []
+        self.log(console, 'Running Split_GenomeSet with params=')
+        self.log(console, "\n" + pformat(params))
+        [OBJID_I, NAME_I, TYPE_I, SAVE_DATE_I, VERSION_I, SAVED_BY_I, WSID_I, WORKSPACE_I, CHSUM_I, SIZE_I, META_I] = list(range(11))  # object_info tuple
```

Comment:
The upper case kind of breaks convention doesn't it?

Reply:
Not sure if we have one in KBase for constants?  I'm used to uppercase for constants.